datacenter/dtr/2.5/guides/admin/configure/deploy-caches/configuration-reference.md

```diff
@@ -0,0 +1,81 @@
+---
+title: DTR cache configuration reference
+description: Learn about the different configuration options for DTR caches.
+keywords: DTR, cache
+---
+
+DTR caches are based on Docker Registry, and use the same configuration
+file format.
+[Learn more about the configuration options](/registry/configuration.md).
+
+The DTR cache extends the Docker Registry configuration file format by
+introducing a new middleware called `downstream` that has three configuration
+options: `blobttl`, `upstreams`, and `cas`:
+
+```none
+# Settings that you would include in a
+# Docker Registry configuration file followed by
+
+middleware:
+  registry:
+      - name: downstream
+        options:
+          blobttl: 24h
+          upstreams:
+            - <Externally-reachable address for upstream registry or content cache in format scheme://host:port>
+          cas:
+            - <Absolute path to next-hop upstream registry or content cache CA certificate in the container's filesystem>
+```
+
+Below you can find the description for each parameter, specific to DTR caches.
+
+<table>
+  <tr>
+    <th>Parameter</th>
+    <th>Required</th>
+    <th>Description</th>
+  </tr>
+  <tr>
+    <td>
+      <code>blobttl</code>
+    </td>
+    <td>
+      no
+    </td>
+    <td>
+The TTL for blobs in the cache. This field takes a positive integer and an optional suffix indicating the unit of time. If
```

Comment:
"The TTL for blobs in the cache." isn't a complete sentence. What did you mean here?

Reply:
I didn't write this content but backported it to `2.5` since the content wasn't archived properly. I will fix though now that we're looking at this. Thanks for the catch, @L-Hudson !